src/decorators.py

```diff
@@ -307,6 +309,7 @@ def caller(self, var, wrapper: MessageDispatcher, message: str):
                 return # commands not allowed in alt channels
 
         if "" in self.commands:
+            assert self.func is not None
```

Comment:
rather than asserting this isn't None everywhere, simply modify the typing of self.func to indicate that it's not Optional